packages/frontend/src/components/ElectionForm/Races/useEditRace.tsx

```diff
@@ -30,7 +30,7 @@ export const useEditRace = (race: iRace | null, race_index: number) => {
         voting_method: 'STAR',
         candidates: [
             { 
-                candidate_id: uuidv4(),
+                candidate_id: makeID(ID_PREFIXES.CANDIDATE, ID_LENGTHS.CANDIDATE),
```

Comment:
Seems like the frontend uses makeID instead of makeUniqueID, but I think these would also need to be unique?